modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/episode/EpisodeContainerFragment.kt

```diff
@@ -265,6 +253,7 @@ class EpisodeContainerFragment :
             multiSelectHelper = bookmarksViewModel.multiSelectHelper,
             menuRes = null,
             activity = requireActivity(),
+            includeStatusBarPadding = false,
```

Comment:
The bottom sheet is handling the status bar padding this fixes the following issue:

| Before | After |
| --- | --- |
| <img width="384" alt="Screenshot 2025-01-28 at 4 18 15 pm" src="https://github.com/user-attachments/assets/66d5147c-0253-49e7-8f2d-d4a9957a5540" /> | <img width="390" alt="Screenshot 2025-01-28 at 4 16 41 pm" src="https://github.com/user-attachments/assets/fede155c-9f3f-4cea-8f52-ef101673aa8f" /> |